ArduCopter/Parameters.h

```diff
@@ -231,7 +231,7 @@ class Parameters {
         //
         // 140: Sensor parameters
         //
-        k_param_imu = 140, // deprecated - can be deleted
```

Comment:
Can't do this.  The comment is very misleading.

You will probably need to do something in g2.